### PR TITLE
Feature/upload to s3

### PR DESCRIPTION
**Pull request by rugeli**
Problem
=======
What is the problem this work solves, including
closing #140 

Solution
========
What I/we did to solve this problem
- In `simularium_helper.py`, added functionalities to store results to S3 - `cellpack-results/simularium` , upload metadata to firebase, and open packing results in simularium app.   

with @meganrm @mogres 

## Type of change
Please delete options that are not relevant.
* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. pack a local recipe: `pack -r [path-to-a-local-recipe] -c [path-to-a-config-file]`
2. pack a remote recipe: pack -r `firebase:recipes/[firebase-recipe-id] -c [path-to-a-config-file]`(make sure you have the recipe living in your firebase before packing)

Keyfiles (delete if not relevant):
-----------------------
1.`simularium_helper.py`
2.`AWSHandler.py`

### CODE REVIEW

File: cellpack/autopack/DBHandler.py [hunk 1]
```diff
@@ -289,18 +330,40 @@ def should_write(self, db):
...
+class DBHandler(object):
+    firebase_handler = None
+
     def __init__(self, db_handler):
         self.db = db_handler
+        if isinstance(db_handler, FirebaseHandler):
+            DBHandler.firebase_handler = db_handler
         self.objects_to_path_map = {}
         self.comp_to_path_map = {}
+        self.grad_to_path_map = {}
```
Comment (meganrm) [inline cellpack/autopack/DBHandler.py:366]: This seems like changes that are outside of the scope of this PR?

**Comment by rugeli** [comment]
This PR has separated into #189 #198 and #201
